interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -637,10 +638,12 @@ public void deadlineExceeded() {
     TestServiceGrpc.TestServiceBlockingStub stub = TestServiceGrpc.newBlockingStub(channel)
         .withDeadlineAfter(10, TimeUnit.MILLISECONDS);
     try {
-      stub.streamingOutputCall(StreamingOutputCallRequest.newBuilder()
-           .addResponseParameters(ResponseParameters.newBuilder()
-               .setIntervalUs(20000))
-               .build()).next();
+      Iterator<StreamingOutputCallResponse> iter = stub.streamingOutputCall(
+          StreamingOutputCallRequest.newBuilder()
+              .addResponseParameters(ResponseParameters.newBuilder().setIntervalUs(20000))
```

Comment:
internalUs does not work that way. It's used between the responses. So for this case, server only sleeps after sending the first response.